SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -22,7 +22,7 @@ const PhysicalWorkerName* SpatialVirtualWorkerTranslator::GetPhysicalWorkerForVi
 
 void SpatialVirtualWorkerTranslator::ApplyVirtualWorkerManagerData(Schema_Object* ComponentObject)
 {
-    UE_LOG(LogSpatialVirtualWorkerTranslator, Log, TEXT("ApplyVirtualWorkerManagerData"));
+    UE_LOG(LogSpatialVirtualWorkerTranslator, Log, TEXT("(%d) ApplyVirtualWorkerManagerData"), LocalVirtualWorkerId);
```

Comment:
Potential driveby (realise this change is just making it saying which worker actually ran this code)
What is this log for? This looks like a specific debugging log that got left in by accident.

Reply:
If this is useful we should probably make the text saying a complete sentence.

Reply:
It doesn't make sense as a single message, but it's a header for the data below, so you get:

(3) ApplyVirtualWorkerManagerData
Translator assignment: 1 - Unreal35238
Translator assignment: 2 - Unreal23412343
etc

So the message just tells you which worker has received the translation mapping and what they think it is.

Reply:
Ah I see. Thanks.